recipes-debian/sources/systemd.inc

```diff
@@ -1,19 +1,19 @@
 # This is generated by debian-source.bbclass
-DPV = "241-7~deb10u3"
+DPV = "241-7~deb10u4"
```

Comment:
I found the CVE-2019-15718.patch in recipes-debian/systemd/systemd/ became no longer necessary. It is resolved in this update.
https://security-tracker.debian.org/tracker/source-package/systemd

So could you also remove the file in this PR?

Reply:
Confirmed. Thanks!

Reply:
I understand, solved this problem.